src/containers/layout/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import List from '@mui/material/List'
+import ListItem from '@mui/material/ListItem'
+
+import HashLink from '~/components/hash-link/HashLink'
+import AppDrawer from '~/components/app-drawer/AppDrawer'
+
+import { styles } from '~/containers/layout/sidebar/Sidebar.styles'
+
+interface SidebarProps {
+  isSidebarOpen: boolean,
+  setIsSidebarOpen: (value:boolean) => void
+  navigationItems:{ route:string, path:string }[]
```

Comment:
may we use this type in another place, maybe will be better to move it to general type?